.circleci/config.yml

```diff
@@ -15,4 +15,6 @@ jobs:
       - run: mix local.hex --force && mix local.rebar --force
       - run: mix deps.get --only test
       - run: mix compile
+      - run: mix hex.build
       - run: mix format --check-formatted
+      - run: mix docs
```

Comment:
Is this just to make sure it doesn't crash?

Reply:
yep